protocol/flows/src/flow_context.rs

```diff
@@ -54,12 +54,17 @@ pub struct FlowContext {
     shared_block_requests: Arc<Mutex<HashSet<Hash>>>,
     transactions_spread: Arc<AsyncRwLock<TransactionsSpread>>,
```

Comment:
`orphans_pool`, `transactions_spread` seem to no longer need to be wrapped into an `Arc`: